pkg/deploy/config.go

```diff
@@ -41,6 +41,7 @@ type Configuration struct {
 	ARMAPICABundle                     *string       `json:"armApiCaBundle,omitempty"`
 	ARMAPIClientCertCommonName         *string       `json:"armApiClientCertCommonName,omitempty"`
 	ARMClientID                        *string       `json:"armClientId,omitempty"`
+	AzSecPackVSATenantId               *string       `json:"azureSecPackVSATenantId,omitempty"`
```

Comment:
What `VSA` stands for?

Reply:
Vulnerability Scanning and Analytics/triage (VSA) it is internal Microsoft team that owns the scanning we are enrolling into.